[2022-08-25 09:06:22,460] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: taxi_rides_data.s3_create_object manual__2022-08-25T09:06:12.106847+00:00 [queued]>
[2022-08-25 09:06:22,479] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: taxi_rides_data.s3_create_object manual__2022-08-25T09:06:12.106847+00:00 [queued]>
[2022-08-25 09:06:22,479] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 09:06:22,479] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-25 09:06:22,479] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 09:06:22,502] {taskinstance.py:1397} INFO - Executing <Task(S3CreateObjectOperator): s3_create_object> on 2022-08-25 09:06:12.106847+00:00
[2022-08-25 09:06:22,515] {standard_task_runner.py:52} INFO - Started process 1255 to run task
[2022-08-25 09:06:22,520] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'taxi_rides_data', 's3_create_object', 'manual__2022-08-25T09:06:12.106847+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/taxi_rides_dag.py', '--cfg-path', '/tmp/tmp9w_kovwx', '--error-file', '/tmp/tmpij139mtv']
[2022-08-25 09:06:22,521] {standard_task_runner.py:80} INFO - Job 53: Subtask s3_create_object
[2022-08-25 09:06:22,625] {task_command.py:371} INFO - Running <TaskInstance: taxi_rides_data.s3_create_object manual__2022-08-25T09:06:12.106847+00:00 [running]> on host ddc8002727e0
[2022-08-25 09:06:59,999] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=taxi_rides_data
AIRFLOW_CTX_TASK_ID=s3_create_object
AIRFLOW_CTX_EXECUTION_DATE=2022-08-25T09:06:12.106847+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-25T09:06:12.106847+00:00
[2022-08-25 09:07:00,145] {base.py:68} INFO - Using connection ID 'aws_credentials' for task execution.
[2022-08-25 09:07:00,147] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-08-25 09:07:57,631] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=taxi_rides_data, task_id=s3_create_object, execution_date=20220825T090612, start_date=20220825T090622, end_date=20220825T090757
[2022-08-25 09:07:57,708] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-25 09:07:57,787] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
